CHANGES.md

```diff
@@ -80,6 +80,8 @@ To be released.
     an empty `TxIds` from peers, and it had made the network waste bandwidth for
     unnecessary messages.  `Swam` became to no more send such empty `GetTxs`.
     [[#297]]
+ -  Reduced the memory footprint of `BlockChain<T>.FindBranchPoint()` method
+    under the circumstances that the height of the `BlockChain<T>` object is high.  [[#282], [#299]]
```

Comment:
Although it's quite trivial, this single line seems too long.  Could you split this line into two?  The maximum width is 80 columns.